source/chip/STM32/STM32L0/Kconfig-chipOptions

```diff
@@ -0,0 +1,1597 @@
+	bool "32"
+	help
+		PLLM == 32
+
+config CHIP_STM32L0_RCC_PLLMUL48
+	bool "48"
+	help
+		PLLM == 48
+
+endchoice
+
+choice
+	prompt "PLL DIV value"
+	default CHIP_STM32L0_RCC_PLLDIV2
+	help
+		PLLDIV value for main PLL. It is used to divide PLL VCO
+		Resulting PLL clock frequency must be in the range:
+		[2MHz; 32MHz].
+
+		PLL clock = PLLVco / PLLDIV = (PLLin * PLLMUL) / PLLDIV
+
+config CHIP_STM32L0_RCC_PLLDIV2
+	bool "2"
+	help
+		PLLN == 2
+
+config CHIP_STM32L0_RCC_PLLDIV3
+	bool "3"
+	help
+		PLLN == 3
+
+config CHIP_STM32L0_RCC_PLLDIV4
+	bool "4"
+	help
+		PLLN == 4
+
+endchoice
+
+endif	# CHIP_STM32L0_RCC_PLL_ENABLE
+
+choice
+	prompt "System clock source"
+	default CHIP_STM32L0_RCC_SYSCLK_PLL if CHIP_STM32L0_RCC_PLL_ENABLE
+	default CHIP_STM32L0_RCC_SYSCLK_HSE if CHIP_STM32L0_RCC_HSE_ENABLE
+	default CHIP_STM32L0_RCC_SYSCLK_HSI16
+	help
+		Select system clock source.
+
+config CHIP_STM32L0_RCC_SYSCLK_HSI16
+	bool "HSI16"
+	help
+		HSI16 oscillator selected as system clock.
+
+config CHIP_STM32L0_RCC_SYSCLK_HSE
+	bool "HSE"
+	depends on CHIP_STM32L0_RCC_HSE_ENABLE
+	help
+		HSE oscillator selected as system clock.
+
+config CHIP_STM32L0_RCC_SYSCLK_PLL
+	bool "PLL"
+	depends on CHIP_STM32L0_RCC_PLL_ENABLE
+	help
+		Main PLL selected as system clock.
+
+config CHIP_STM32L0_RCC_SYSCLK_MSI
+	bool "MSI"
+	depends on CHIP_STM32L0_RCC_MSI_ENABLE
+	help
+		MSI oscillator selected as system clock.
+
+endchoice
+
+endif	# CHIP_STM32L0_STANDARD_CLOCK_CONFIGURATION_ENABLE
+
+if !CHIP_STM32L0_STANDARD_CLOCK_CONFIGURATION_ENABLE
+
+config CHIP_STM32L0_RCC_SYSCLK_FREQUENCY
+	int "Frequency of SYSCLK, Hz"
+	range 1 32000000
+	default 8000000
+	help
+		Frequency of SYSCLK, Hz. RCC must be configured by user to achieve that
+		frequency.
+
+endif	# !CHIP_STM32L0_STANDARD_CLOCK_CONFIGURATION_ENABLE
+
+choice
+	prompt "AHB divider"
+	default CHIP_STM32L0_RCC_AHB_DIV1
+	help
+		AHB clock division factor
+
+		AHBclk = SYSclk / AHBdivider
+
+config CHIP_STM32L0_RCC_AHB_DIV1
+	bool "1"
+	help
+		System clock not divided
+
+config CHIP_STM32L0_RCC_AHB_DIV2
+	bool "2"
+	help
+		System clock divided by 2
+
+config CHIP_STM32L0_RCC_AHB_DIV4
+	bool "4"
+	help
+		System clock divided by 4
+
+config CHIP_STM32L0_RCC_AHB_DIV8
+	bool "8"
+	help
+		System clock divided by 8
+
+config CHIP_STM32L0_RCC_AHB_DIV16
+	bool "16"
+	help
+		System clock divided by 16
+
+config CHIP_STM32L0_RCC_AHB_DIV64
+	bool "64"
+	help
+		System clock divided by 64
+
+config CHIP_STM32L0_RCC_AHB_DIV128
+	bool "128"
+	help
+		System clock divided by 128
+
+config CHIP_STM32L0_RCC_AHB_DIV256
+	bool "256"
+	help
+		System clock divided by 256
+
+config CHIP_STM32L0_RCC_AHB_DIV512
+	bool "512"
+	help
+		System clock divided by 512
+
+endchoice
+
+choice
+	prompt "APB1 (low speed) divider"
+	default CHIP_STM32L0_RCC_APB1_DIV2
+	help
+		APB1 (low speed) clock division factor. APB1 clock must not exceed:
+		- 42MHz for STM32L001, STM32L005, STM32L007, STM32L015 and STM32L017
```

Comment:
Do such chips (L001) exist? This looks like a copy + replace from some other family (STM32F4) (; For STM32L0 max frequency of APB is 32MHz for all chips and for both APBs. Due to that fact I think that while we're at it, the default values can be adjusted too, along with default values for PLL to get 32MHz clock right away (;